ubuntu_18.04-dev.dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:18.04
```

Comment:
we can set an argument for the ubuntu version number

Reply:
I think we can just go with 18.04

Reply:
agreed, just use 18.04 now.